src/ray/gcs/gcs_server/gcs_server.cc

```diff
@@ -484,6 +483,16 @@ void GcsServer::InstallEventListeners() {
         gcs_actor_manager_->SchedulePendingActors();
       });
     });
+
+    auto gcs_actor_scheduler =
+        std::dynamic_pointer_cast<GcsBasedActorScheduler>(gcs_actor_scheduler_);
+    gcs_actor_scheduler->AddClusterResourcesChangedListener([this] {
```

Comment:
Should the callback the same for AddClusterResourcesChangedListener && AddResourcesChangedListener here?

Reply:
Fixed